deploy/Datadog.Trace.ClrProfiler.WindowsInstaller/Product.wxs

```diff
@@ -139,31 +140,45 @@
       </Component>
 
       <Component Win64="$(var.Win64)">
-        <File Id="integrations.json" Source="..\..\integrations.json" />
+        <File Id="integrations.json" Source="$(var.TracerHomeDirectory)\integrations.json" />
       </Component>
     </ComponentGroup>
 
-    <ComponentGroup Id="Files.Native" Directory="INSTALLFOLDER">
-      <Component Win64="$(var.Win64)">
+    <?if $(var.Win64) = yes ?>
+    <ComponentGroup Id="Files.Native.64" Directory="INSTALLFOLDER">
+      <Component Win64="yes">
         <File Id="Datadog.Trace.ClrProfiler.Native"
-              Source="$(var.NativeDllPath)\Datadog.Trace.ClrProfiler.Native.dll"
+              Source="$(var.TracerHomeDirectory)\win-x64\Datadog.Trace.ClrProfiler.Native.dll"
               Checksum="yes">
           <Class Id="$(var.ProfilerCLSID)" Context="InprocServer32" ThreadingModel="both" Description="$(var.ProductNamePlatformAgnostic)"/>
         </File>
       </Component>
     </ComponentGroup>
 
     <!-- For the 64-bit installer, also install the 32-bit profiler -->
-    <?if $(var.Win64) = yes ?>
     <ComponentGroup Id="Files.Native.32" Directory="INSTALLFOLDER.32">
```

Comment:
I brought this up recently and could be added to this PR: Can we just drop this into the same output `INSTALLFOLDER` instead of `INSTALLFOLDER.32`? That way contents are only placed onto the drive requested by the user.

Reply:
I would like to do that, too. I think I tried a while ago, but it created a conflict for wix: if the file is marked with `Win64=no`, we can't install it here, but if it's marked with `Win64=yes`, then we can't register as a 32-bit COM component.

An alternative could be to stop using the registry to register the profiler as COM and instead use `COR[ECLR]_PROFILER_PATH_64`/`COR[ECLR]_PROFILER_PATH_32`, but I don't know when support for those platform-specific variable were added to .NET Framework.

Reply:
Okay. I don't want to de-rail this PR just for this change so we can shelve this for another PR. Anyways, it's probably better if the PR has the sole focus of refactoring and not changing functionality